drivers/gamatronic.c

```diff
@@ -358,11 +359,12 @@ void setup_serial(const char *port)
     if (i == 5) {
 	printf("Can't talk to UPS on port %s!\n",port);
 	printf("Check the cabling and portname and try again\n");
-	printf("Please note that this driver only support UPS Models with SEC Protorol\n");
+	printf("Please note that this driver only support UPS Models with SEC Protocol\n");
```

Comment:
yup:
- indent is bad and mixed in this driver, and should be revamped (iirc, you have something in progress, no?)
- the else is indeed just to quiet down cppcheck